File: server/utils/seedLoader.mjs
import { readFileSync } from 'node:fs';
import path from 'node:path';
import { fileURLToPath } from 'node:url';

const __filename = fileURLToPath(import.meta.url);
const __dirname = path.dirname(__filename);
const DATA_DIR = path.join(__dirname, '..', 'data');

export function readSeedJson(name) {
  const p = path.join(DATA_DIR, name);
  return JSON.parse(readFileSync(p, 'utf-8'));
}

File: backend/app/utils/seed_loader.py
from __future__ import annotations

import json
from pathlib import Path
from typing import Any

DATA_DIR = Path(__file__).resolve().parents[3] / "backend" / "data"


def read_seed_json(name: str) -> dict[str, Any]:
    with (DATA_DIR / name).open("r", encoding="utf-8") as f:
        return json.load(f)
